packages/core/src/lib/relay/index.ts

```diff
@@ -61,8 +62,7 @@ class Relay extends GossipSub implements IRelay {
       fallbackToFloodsub: false,
     });
 
-    super(components, options);
-    this.multicodecs = constants.RelayCodecs;
```

Comment:
I expect this to  still be needed on `this.gossipSub`.

Reply:
Should be ok now